src/pages/Before.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { change } from '../redux/slice/emotionSlice';
+import weekmotion from '../assets/images/weekmotion.svg';
+import { positiveEmotion, negativeEmotion } from '../assets/strings/emotions';
+
+export default function Before() {
+  const [emotion, setEmotion] = useState<string[]>([]);
+  const dispatch = useDispatch();
+
+  const onClick = (event: React.MouseEvent<HTMLDivElement>) => {
+    const { innerText } = event.target;
```

Comment:
별건 아닌데, 린트에서 에러가 나서 넣어봅니다
```suggestion
    const { innerText } = event.target as HTMLDivElement;
```